README.md

```diff
@@ -45,13 +59,37 @@ yarn scripts:addTrip
 
 You can run a migration service to add aggregation data on your timeseries. This is necessary because the trips documents can be huge and negatively impact the app performances. Therefore, we rely on an aggregated trip view on the app side. If the aggregation is missing, the trip won't be displayed.
 
+⚠️ Make sure to update your development URL in `./konnector-dev-config.json` in the field `"COZY_URL"` if you do not wish to use the default URL `http://cozy.localhost:8080`.
+
+⚠️ Your instance database must contain in its `io.cozy.accounts` the following document as-is in order to be able to find the `geoseries` collections, please insert it manually or with a mango request into the associated database:
```

Comment:
cette partie concerne les fixtures non ? C'est un script manquant qu'on devrait plutôt ajouter côté ACH via la commande `yarn fixtures` non ?

Reply:
C'est très propre à coach cette partie. CCo2 appelle `ach import data`. De là, ACH n'a aucune idée qu'il faut avoir cette information. 

Il faudrait modifier les fixtures de CCo2 pour ajouter ce document directement. Comme ça, `yarn fixtures` insère ça tout seul comme un grand sans avoir besoin de le faire à la main;

Reply:
les fixtures sont gérées côté ACH https://github.com/cozy/ACH/blob/master/scripts/timeseries/importGeojson.js et https://github.com/cozy/ACH/blob/master/data/accounts/tracemob.json et non côté CCO2 🤔

Reply:
Oui ça le fait déjà, mis en place par un certain @JF-Cozy :D

Reply:
@acezard Can you run `ACH --version` please? I'm wondering if you have not an old version of `ACH` since this `account` stuff was added later. (since 1.39 https://github.com/cozy/ACH/commit/91ac48f2d3cfc66c67de6e5368e9f5b40f4a8a26). 

If you have an old version it can explain the diff (and also why it's a bad idea to have fixtures for an app inside another repo), and it should be fixed by using `yarn ACH fixtures` instead of `ACH fixtures`

Reply:
1.48.0

Reply:
So I don't understand why the account was not created. 

I tried to run the fixtures, here is what I've encountered: 

```
➜  coachCO2 git:(master) yarn fixtures -u http://q.cozy.localhost:8080
yarn run v1.22.19
$ ACH import accounts/tracemob.json && ACH script timeseries/importGeojson -u http://q.cozy.localhost:8080
Could not automatically create token
Could not connect to http://cozy.localhost:8080
FetchError: {"error":{}}
```

=> Always a bad idea to have several commands in one since args are not well used. So `yarn fixtures` will only works for `http://cozy.localhost:8080` which is not really the real default anymore. So we should have 2 commands one for tracemob and one for importeGeojson. Or we should have one that deal correctly with the args.

By running manually the commands, I have: 
```
➜  coachCO2 git:(master)  ACH import accounts/tracemob.json -u http://q.cozy.localhost:8080
Could not automatically create token
Importing 1 documents for doctype io.cozy.accounts...
io.cozy.accounts: 100.00%
Imported 1 io.cozy.accounts document
➜  coachCO2 git:(master) ACH script timeseries/importGeojson -u http://q.cozy.localhost:8080
Launching script timeseries/importGeojson...
Dry run : true
Import 75 trips...
Done!
```

So it seems that account and trips are well imported. @acezard do you have any issue with the first command? 


After that I launched the app and then nothing was displayed: 
![Capture d’écran 2023-05-10 à 14 38 18](https://github.com/cozy/coachCO2/assets/1107936/9fd43ac2-9012-4306-b85f-876b0b8451c4)

data/io.cozy.accounts/_find => OK 1 result returned 
data/io.cozy.timeseries.geojson/_find => KO 0 result returned 

Then I read the readme and saw that I had to run a script.

```
➜  coachCO2 git:(master) yarn scripts:addTrip -u http://q.cozy.localhost:8080
yarn run v1.22.19
$ node scripts/save-random-trip.js -u http://q.cozy.localhost:8080
usage: save-random-trip.js [-h] [--source-account SOURCE_ACCOUNT] [--url URL]
save-random-trip.js: error: Unrecognized arguments: -u http://q.cozy.localhost:8080.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

OK -u is not supported here while its supported with ACH

```
➜  coachCO2 git:(master) yarn scripts:addTrip --url http://q.cozy.localhost:8080
yarn run v1.22.19
$ node scripts/save-random-trip.js --url http://q.cozy.localhost:8080
{"time":"2023-05-10T12:42:01.831Z","type":"debug","namespace":"create-cli-client","message":"Starting OAuth flow"}
{"time":"2023-05-10T12:42:01.953Z","type":"debug","namespace":"create-cli-client","message":"OAuth callback server started, waiting for authentication"}
{"time":"2023-05-10T12:42:04.030Z","type":"debug","namespace":"create-cli-client","message":"Authenticated, Shutting server down"}
{"time":"2023-05-10T12:42:04.378Z","type":"debug","namespace":"create-cli-client","message":"Saving credentials to /tmp/cozy-client-oauth-q-cozy-localhost:8080-io.cozy.client.cli-1930758627.json"}
Trip added: 45a2907ca806f82d2117b88488063afd - from: 2023-05-10T12:42:04.415Z - to: 2023-05-10T12:42:04.415Z of 0 km in PredictedModeTypes.TRAIN
✨  Done in 3.21s.
```

Reloaded the page => still nothing to show. 

data/io.cozy.timeseries.geojson/_find => still returning 0 doc.

Reply:
tu n'as pas à exécuter le script AddTrip, cela semble être un `ou` : `It is also possible to generate a random trip`. Je ne sais pas pourquoi on a ajouté ce script en plus des fixtures, cc @paultranvan . Peut-être pour éviter de devoir lancer le script de migration `timeseriesWithoutAggregateMigration` après l'ajout des fixtures ?